### DISCUSSION TITLE
Hi I'm actually new here please how do I get followers?

### DISCUSSION THREAD

**Discussion by Stephensamaila** [community/community#135272] [New to GitHub] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hi friend I'm new here

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Stephensamaila ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Stephensamaila** [reply]
Thanks for your reply ❤️

**Reply by datodato788** [reply]
d

**Reply by ArvizuAutoCenter** [reply]
[0_dump_all.zip](https://github.com/user-attachments/files/16577446/0_dump_all.zip)

**Reply by ArvizuAutoCenter** [reply]
Need to remover divices,block,this people

**Reply by TheGiraffe3** [reply]
Please do not spam community discussions, @ArvizuAutoCenter and @datodato788.

**Comment by datodato788** [comment]
w

**Reply by datodato788** [reply]
w

**Reply by Stephensamaila** [reply]
Hello dato nice to meet you here😊

**Comment by TheGiraffe3** [comment]
Someone else has to go to your profile and click the "Follow" button.
But GitHub isn't really a _social_ platform. Somebody following you normally means that they want to see what you are doing.

**Comment by tbrowne5** [comment]
Hi Julie
Welcome to github, I hope your coding journey is going well!
When it comes to gaining followers, the only true method is to contribute to public projects and discussions. If you can provide useful edits to open source code, people will appreciate that. If you give meaningful answers to questions asked in the discussion section, people may follow you. Finally, if you are working on interesting public projects, people will pay attention.
Best of luck!
Tyler

**Comment by a01jafary** [comment]
Someone else has to go to your profile and click the "Follow" button.

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Stephensamaila 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
